tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathDescriptions.txt

```diff
@@ -193,14 +193,14 @@
-^//ldml/dates/fields/field\[@type="([^"]*)"]/relativeTime\[@type="future"]/relativeTimePattern\[@count="([^"]*)"]; Provide a pattern used to display times in the future. For more information, please see %DATE_TIME_NAMES.
-^//ldml/dates/fields/field\[@type="([^"]*)"]/relativeTime\[@type="past"]/relativeTimePattern\[@count="([^"]*)"]; Provide a pattern used to display times in the past. For more information, please see %DATE_TIME_NAMES.
+^//ldml/dates/fields/field\[@type="day"]/relative\[@type="0"]; Provide the name for today. For more information, please see %DATE_TIME_NAMES_RELATIVE. The lettercasing should be appropriate for the top example. If the lettercasing is then wrong for the bottom example, please file a ticket to fix contextTransforms/relative/stand-alone.
+^//ldml/dates/fields/field\[@type="day"]/relative\[@type="-([^"]*)"]; Provide a name for the day, {1} before today. For more information, please see %DATE_TIME_NAMES_RELATIVE. The lettercasing should be appropriate for the top example. If the lettercasing is then wrong for the bottom example, please file a ticket to fix contextTransforms/relative/stand-alone.
+^//ldml/dates/fields/field\[@type="day"]/relative\[@type="([^"]*)"]; Provide a name for the day, {1} after today. For more information, please see %DATE_TIME_NAMES_RELATIVE. The lettercasing should be appropriate for the top example. If the lettercasing is then wrong for the bottom example, please file a ticket to fix contextTransforms/relative/stand-alone.
+^//ldml/dates/fields/field\[@type="([^"]*)"]/relative\[@type="0"]; Provide the name for “this {2}”. For more information, please see %DATE_TIME_NAMES_RELATIVE. The lettercasing should be appropriate for the top example. If the lettercasing is then wrong for the bottom example, please file a ticket to fix contextTransforms/relative/stand-alone.
+^//ldml/dates/fields/field\[@type="([^"]*)"]/relative\[@type="-1"]; Provide a name for “last {1}”. For more information, please see %DATE_TIME_NAMES_RELATIVE. The lettercasing should be appropriate for the top example. If the lettercasing is then wrong for the bottom example, please file a ticket to fix contextTransforms/relative/stand-alone.
+^//ldml/dates/fields/field\[@type="([^"]*)"]/relative\[@type="1"]; Provide a name for “next {1}”. For more information, please see %DATE_TIME_NAMES_RELATIVE. The lettercasing should be appropriate for the top example. If the lettercasing is then wrong for the bottom example, please file a ticket to fix contextTransforms/relative/stand-alone.
+^//ldml/dates/fields/field\[@type="([^"]*)"]/relativeTime\[@type="future"]/relativeTimePattern\[@count="([^"]*)"]; Provide a pattern used to display times in the future. For more information, please see %DATE_TIME_NAMES. The lettercasing should be appropriate for the top example. If the lettercasing is then wrong for the bottom example, please file a ticket to fix contextTransforms/relative/stand-alone.
```

Comment:
good, i've conflicted with it in #4633 but that's OK

Reply:
Sorry about that